test/testenv/verificationutils.go

```diff
@@ -569,3 +569,37 @@ func VerifyAppFrameworkDeployment(standalone *enterprisev1.Standalone) {
 	}
 
 }
+
+// VerifyAppInstall verify that app of specific version is installed
+func VerifyAppInstall(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, apps map[string]string, versionCheck bool, statusCheck string) {
+	for appName, version := range apps {
+		status, versionInstalled, err := GetPodAppStatus(deployment, podName, ns, appName)
+		gomega.Expect(err).Should(gomega.Equal(""))
```

Comment:
Nit: This can be failed like this `gomega.Expect(err).To(Succeed(), "Unable to get app status on pod ")`
This way we fail with meaningful error.